Initium-Odp/src/com/universeprojects/miniup/server/scripting/wrappers/Item.java

```diff
@@ -58,6 +58,23 @@ public boolean setContainer(EntityWrapper ent)
 		return true;
 	}
 	
+	private Item[] contents = null;
+	public Item[] getContents() {
+		if(contents != null) return contents;
+		
+		List<CachedEntity> results = db.getFilteredList("Item", "containerKey", getKey());
+				
+		List<Item> toReturn = new ArrayList<>();
+		
+		for(CachedEntity ce : results) {
+			toReturn.add((Item) ScriptService.wrapEntity(ce, db));
+		}
+		
+		contents = toReturn.toArray(new Item[toReturn.size()]);
+		
+		return contents;
+	}

```

Comment:
Make sure that the parent is not also an item, since we don't want to allow querying nested containers at all (to mimic the limitations of the ODP, don't want to create a workaround in scripting).